packages/angular_devkit/core/src/json/schema/registry.ts

```diff
@@ -361,57 +338,32 @@ export class CoreSchemaRegistry implements SchemaRegistry {
 
       // Validate using ajv
       const result = validator.call(validationContext, data);
```

Comment:
I think this can also still return a promise.  Prompts should be the scenario where this can happen.

Reply:
Probably could restructure this section with an `await` on the call itself to normalize the result and then wrapped in a `try`/`catch` to normalize the different error handling mechanisms (`validator.errors` vs the exception).

Reply:
You are right.